gridstatus/ercot.py

```diff
@@ -646,6 +710,63 @@ def _finalize_spp_df(df, settlement_point_field, locations):
         df = df.reset_index(drop=True)
         return df
 
+    @staticmethod
+    def _finalize_as_price_df(doc, timezone, ambiguous_column, pivot=False):
```

Comment:
you shouldn't need to pass `timezone` since you can just use `Ercot.default_timezone`

Reply:
resolved using suggestion